mock-core-configs/etc/mock/templates/kylin-10.tpl

```diff
@@ -4,9 +4,8 @@ config_opts['releasever'] = '10'
 config_opts['package_manager'] = 'dnf'
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
 
-# No official up to date image available, the existing ones are all outdated:
-# https://cr.kylinos.cn/zh/image?name=kylin-server
-config_opts['use_bootstrap_image'] = False
+# Registry web interface: https://cr.kylinos.cn/zh/image
+config_opts['bootstrap_image'] = 'cr.kylinos.cn/kylin/kylin-server-platform:v10sp3-2403'
```

Comment:
Isn't there a better tag? I'm concerned that we'll have to update this reference frequently in the future.

Reply:
Ack, there's no other option then.

Reply:
They might do that, at one point. I will update if they do.